client/hotrod-client/src/test/java/org/infinispan/client/hotrod/admin/RemoteCacheAdminTest.java

```diff
@@ -96,9 +107,25 @@ public void cacheCreateRemoveTest(Method m) {
       assertFalse(manager(1).cacheExists(cacheName));
    }
 
+   public void cacheCreateMissingTemplate(Method m) throws InterruptedException {
+      Exceptions.expectException(HotRodClientException.class,
+                                 () -> client(0).administration().withFlags(CacheContainerAdmin.AdminFlag.VOLATILE)
+                                                .createCache("some_name", "some_template_name"));
+      Thread.sleep(999);
```

Comment:
Is this sleep needed?

Reply:
If so maybe we change the socket timeout to be lower and lower this as well?

Reply:
Updated to remove the sleep and check the task queue in the scheduled executor instead @wburns